features/forced-colors.yml

```
@@ -0,0 +1,14 @@
+name: Forced colors
+description: The `forced-colors` CSS @media rule detects when a user has chosen to use a forced colors mode, also known as high-contrast mode, and the `forced-color-adjust` CSS property sets whether force colors apply to an element.
```

I'll apply this fix myself and merge.

```suggestion
description: The `forced-colors` CSS @media rule detects when a user has chosen to use a forced colors mode, also known as high-contrast mode, and the `forced-color-adjust` CSS property sets whether forced colors apply to an element.
```